### PR TITLE
Fix mismatch between defaultLocalizationPose in frontend and backend and use RobotResponse in RobotController

### PR DESCRIPTION
**Pull request by prasm313**

### CODE REVIEW

File: backend/api/Controllers/RobotController.cs [hunk 1]
```diff
@@ -290,8 +297,9 @@ public async Task<ActionResult<IList<VideoStream>>> GetVideoStreams([FromRoute]
                 _logger.LogWarning("Could not find robot with id={Id}", robotId);
                 return NotFound();
             }
+            var robotResponse = new RobotResponse(robot);
 
-            return Ok(robot.VideoStreams);
+            return Ok(robotResponse.VideoStreams);
```
Comment (Eddasol) [inline backend/api/Controllers/RobotController.cs:302]: This change is unnecessary. Suggest reverting to how it was

File: backend/api/Services/RobotService.cs [hunk 2]
```diff
@@ -173,6 +173,12 @@ private IQueryable<Robot> GetRobotsWithSubModels()
                 .Include(r => r.VideoStreams)
                 .Include(r => r.Model)
                 .Include(r => r.CurrentArea)
+                .ThenInclude(r => r != null ? r.Deck : null)
+                .Include(r => r.CurrentArea)
```
Comment (UsamaEquinorAFK) [inline backend/api/Services/RobotService.cs:177]: is this supposed to be r.Deck ? :) Same for the other lines below

File: backend/api/Services/RobotService.cs [hunk 3]
```diff
@@ -173,6 +173,12 @@ private IQueryable<Robot> GetRobotsWithSubModels()
...
+                .ThenInclude(r => r != null ? r.Plant : null)
+                .Include(r => r.CurrentArea)
```
Comment (UsamaEquinorAFK) [inline backend/api/Services/RobotService.cs:179]: r.Plant

File: backend/api/Services/RobotService.cs [hunk 4]
```diff
@@ -173,6 +173,12 @@ private IQueryable<Robot> GetRobotsWithSubModels()
...
+                .ThenInclude(r => r != null ? r.Installation : null)
+                .Include(r => r.CurrentArea)
```
Comment (UsamaEquinorAFK) [inline backend/api/Services/RobotService.cs:181]: r.Installation ?

File: backend/api/Services/RobotService.cs [hunk 5]
```diff
@@ -173,6 +173,12 @@ private IQueryable<Robot> GetRobotsWithSubModels()
                 .Include(r => r.VideoStreams)
                 .Include(r => r.Model)
                 .Include(r => r.CurrentArea)
+                .ThenInclude(r => r != null ? r.Deck : null)
```
Comment (Eddasol) [inline backend/api/Services/RobotService.cs:176]: Use something more descriptive than r
```suggestion
                .ThenInclude(currentArea => currentArea != null ? currentArea.Deck : null)
```

File: backend/api/Services/RobotService.cs (hunk 2 above)
Comment (prasm313) [reply backend/api/Services/RobotService.cs:177]: Changed name to area

**Review by Eddasol — APPROVED** [review]
LGTM :)

**Pull request merged**